neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
Já jsem to tam nepřidal, takže jinam to taky dodělávat nebudu. A připadá mi to informativní, takže bych to nemazal.